packages/luvi/build.sh

```diff
@@ -0,0 +1,52 @@
+TERMUX_PKG_HOMEPAGE=https://luvit.io
+TERMUX_PKG_DESCRIPTION="A project in-between luv and luvit."
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="Komo @cattokomo"
+TERMUX_PKG_VERSION=20240216
+_commit=b85da58f2549a519486a7296f5b836a8f6a64880
+_version=2.14.0
+TERMUX_PKG_GIT_BRANCH=master
+TERMUX_PKG_SRCURL=git+https://github.com/luvit/luvi.git
+TERMUX_PKG_DEPENDS="pcre, openssl, luv, lua51-lpeg, libluajit"
+TERMUX_PKG_BUILD_DEPENDS="luv, lua51-lpeg"
+TERMUX_PKG_SUGGESTS="lit, luvit"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
+  -DWithSharedLibluv=On
+  -DWithOpenSSL=On
+  -DWithSharedOpenSSL=On
+  -DWithPCRE=On
+  -DWithSharedPCRE=On
+  -DWithLPEG=On
+  -DWithSharedLPEG=On
+"
+
+termux_step_post_get_source() {
+	git checkout "${_commit}"
+}
+
+termux_step_pre_configure() {
+	declare include=${TERMUX_PREFIX}/include lib=${TERMUX_PREFIX}/lib
+	TERMUX_PKG_EXTRA_CONFIGURE_ARGS+="
+    -DLIBLUV_INCLUDE_DIR=${TERMUX_PKG_SRCDIR}/deps/luv/src
+    -DLIBLUV_LIBRARIES=$lib/libluv.so
+    -DLUAJIT_INCLUDE_DIR=$include/luajit-2.1
+    -DLUAJIT_LIBRARIES=$lib/libluajit.so
+    -DLIBUV_INCLUDE_DIR=$include
+    -DLIBUV_LIBRARIES=$lib/libuv.so
+    -DOPENSSL_INCLUDE_DIR=$include
+    -DOPENSSL_LIBRARIES=$lib
+    -DPCRE_INCLUDE_DIR=$include
+    -DPCRE_LIBRARIES=$lib
+  "
+
+	TARGET_ARCH=${TERMUX_ARCH}
+	case "${TARGET_ARCH}" in
```

Comment:
If we're testing `TERMUX_ARCH`, why assign its value to `TARGET_ARCH`?
```suggestion
	case "${TERMUX_ARCH}" in
```

Reply:
`TARGET_ARCH` is used to specify the arch for compiled LuaJIT bytecodes. See `cmake/Modules/LuaJITAddExecutable.cmake` in line 12 to line 14.

Reply:
Yes I get that.
But it seems more sensible to me to check the case of `TERMUX_ARCH`, and assign the appropriate `TARGET_ARCH` for ***all*** architectures.
Even if `arm` is already valid.
Instead of reassigning `TARGET_ARCH` in 3/4 cases.